Backend/sub_crates/str_util/src/tests/strformat.rs

```diff
@@ -1,11 +1,37 @@
 extern crate proptest;
-use crate::strformat;
 use self::proptest::prelude::*;
+use crate::strformat;
+
+proptest! {
+  /// This test is supplied with three random inputs:
+  ///
+  /// 1. Size of the generated arguments, in interval `[0,32)`
```

Comment:
Small detail: Isn't the 32 included as a possible size in the argument generation `size in 0usize..33usize`?

Reply:
Fixed